core/templates/dev/head/pages/exploration_editor/param_changes_editor_directive.html

```diff
@@ -47,9 +47,12 @@
       </span>
       <span>
         Change
-        <select2-dropdown item="paramChange.name" choices="paramNameChoices"
-                          placeholder="Param name" new-choice-regex="^[A-Za-z]+$"
-                          width="143px">
+        <select2-dropdown item="paramChange.name"
+                          choices="paramNameChoices"
+                          placeholder="Param name"
+                          new-choice-regex="^[A-Za-z]+$"
+                          width="143px"
+                          tag-mode="true">
```

Comment:
From my observation, the meaning of tags changed a little bit in the new version of select2. The tags mean that new option can be added, that's precisely what we need in parameters. 

I tested the parameters and I found only one odd thing when there is text input interaction and the user input should be same or contain some parameter, the text input doesn't accept the parameter value but the exact string.
For example, parameter item is "something" and user input should equal to `{{item}}`, if the user enters "something" then nothing happens, but when the user enters "{{item}}" then it's evaluated as correct input.
But I found this "bug" in develop branch too.